src/index.js

```diff
@@ -89,6 +91,15 @@ export default class TronWeb extends EventEmitter {
 
         this.fullnodeVersion = DEFAULT_VERSION;
         this.injectPromise = injectpromise(this);
+
+        this.tronGrid = new TronGrid(this);
+        this.useTronGridAPI = options.useTronGridAPI || false;
+    }
+
+    useTrongrid(options = {}) {
+        if (typeof options.useTronGridAPI === 'boolean')
+            return options.useTronGridAPI;
+        return this.useTronGridAPI;
```

Comment:
is this.useTronGridAPI  variable really necessary?  We could just return false as default value if options.useTronGridAPI is not a boolean.